polkadot/xcm/xcm-executor/src/assets.rs

```diff
@@ -21,16 +21,16 @@ use sp_std::{
 	prelude::*,
 };
 use xcm::latest::{
-	AssetId, AssetInstance,
+	Asset, AssetFilter, AssetId, AssetInstance, Assets,
 	Fungibility::{Fungible, NonFungible},
-	InteriorMultiLocation, MultiAsset, MultiAssetFilter, MultiAssets, MultiLocation,
+	InteriorLocation, Location,
+	WildAsset::{All, AllCounted, AllOf, AllOfCounted},
 	WildFungibility::{Fungible as WildFungible, NonFungible as WildNonFungible},
-	WildMultiAsset::{All, AllCounted, AllOf, AllOfCounted},
 };
 
 /// List of non-wildcard fungible and non-fungible assets.
 #[derive(Default, Clone, RuntimeDebug, Eq, PartialEq)]
-pub struct Assets {
+pub struct HoldingAssets {
```

Comment:
```suggestion
pub struct AssetsInHolding {
```

sounds like a better name. In addition, the comment above this struct needs to be updated.